browser/ui/android/omnibox/java/src/org/chromium/chrome/browser/omnibox/suggestions/editurl/BraveEditUrlSuggestionProcessor.java

```diff
@@ -48,4 +49,56 @@ public boolean doesProcessSuggestion(@NonNull AutocompleteMatch suggestion, int
 
         return super.doesProcessSuggestion(suggestion, position);
     }
+
+    /*
+     * Compared to upstream, we have different behaviour on clicking url bar https://github.com/brave/brave-browser/issues/10524
```

Comment:
why do we have different behavior? I don't think we have different behavior on desktop? Shouldn't it be consistent?